hack/make-rules/make-help.sh

```diff
@@ -21,22 +21,25 @@ set -o pipefail
 readonly red=$(tput setaf 1)
 readonly reset=$(tput sgr0)
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 ALL_TARGETS=$(make -C "${KUBE_ROOT}" PRINT_HELP=y -rpn | sed -n -e '/^$/ { n ; /^[^ .#][^ ]*:/ { s/:.*$// ; p ; } ; }' | sort)
-CMD_TARGETS=$(ls -l "${KUBE_ROOT}/cmd" |awk '/^d/ {print $NF}')
+CMD_TARGETS=$(find . "${KUBE_ROOT}/cmd" |awk '/^d/ {print $NF}')
```

Comment:
Why did it change and what did it fix:
```
In ./hack/make-rules/make-help.sh line 26:
CMD_TARGETS=$(ls -l "${KUBE_ROOT}/cmd" |awk '/^d/ {print $NF}')
              ^----------------------^ SC2012: Use find instead of ls to better handle non-alphanumeric filenames.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2012
```

Reply:
this is not equivalent. shellcheck is correct to switch to `find`, but the change here is not the same.

Reply:
The line outputs directory list under ${KUBE_ROOT}/cmd. How about the following?
```
CMD_TARGETS=$(find "${KUBE_ROOT}/cmd" -maxdepth 1 -mindepth 1 -type d | sed "s|${KUBE_ROOT}\/cmd\/||" | sort)
```